scripts/trufflehog-check.sh

```diff
@@ -15,19 +15,30 @@ fi
 
 echo "trufflehog-check.sh: Scanning repo ..."
 
-last_merge=$(git log --format=format:"%H" -n 1 raft-tdp-main)
-
 # $1 - The first argument to this script, the current git branch name
+CIRCLE_BRANCH=$1
+
+# If the current PR branch includes `raft-tdp-main`, it a merge against HHS:main,
+# so we update the target branch to scan against.
+if [ "$CIRCLE_BRANCH" == "raft-tdp-main" ] ; then
+  TARGET=main
+else
+  TARGET=raft-tdp-main
+fi
+
+# Get the hash of the latest commit in the target branch.
+last_merge=$(git log --format=format:"%H" -n 1 $TARGET)
+
 # --since_commit - Look at all commits since the last merge into raft-tdp-main
 # --entropy=True - Entropy checks on large git diffs
 python ./trufflehog-check/lib/python3.8/site-packages/truffleHog/truffleHog.py \
   --regex \
   --entropy=True \
-  --branch "$1" \
+  --branch "$CIRCLE_BRANCH" \
   --since_commit "$last_merge" \
   --exclude_paths ./trufflehog-exclude-patterns.txt \
   --rules ./regexes.json \
-  https://github.com/raft-tech/TANF-app
+  file:///root/project
```

Comment:
Updated this to scan the locally cloned repo, so we don't have to specify a given repository by url.